### ISSUE TITLE
`udp_multicast_join` test failure on `osx_64` `libuv v1.49.1`

### ISSUE THREAD

**Issue by jjerphan** [libuv/libuv#4574]
* **Version**: v1.49.1
* **Platform**: `osx_64` 

From https://github.com/conda-forge/libuv-feedstock/pull/75 (see [those logs](https://github.com/conda-forge/libuv-feedstock/pull/75#issuecomment-2407398762)):
```
not ok 407 - udp_multicast_join
# exit code 6
# Output from process `udp_multicast_join`:
# Assertion failed in test/test-udp-multicast-join.c on line 131: `r == 0` (-16 == 0)
```

**Comment by saghul** [comment]
What macOS version are you using? That is known to be broken in macOS > 12. I need to get back to skipping that test there...

**Comment by jjerphan** [comment]
macOS 13 is used, namely [the `macOS 13 (20241008)` image](https://github.com/actions/runner-images/releases/tag/macos-13%2F20241008.186).

**Comment by bnoordhuis** [comment]
This was fixed by a3abfbcb088e639acddfed2771d85426c3392fe9, right? Not yet released but you can cherry-pick it if you want. You're also going to need the preceding commit.
